tests/PolykeyClient.test.ts

```diff
@@ -77,11 +81,16 @@ describe('PolykeyClient', () => {
     expect(await session.readToken()).toBeDefined();
     await pkClient.destroy();
     expect(await session.readToken()).toBeUndefined();
-    await webSocketClient.destroy({ force: true });
   });
   test('end to end with authentication logic', async () => {
-    const webSocketClient = await WebSocketClient.createWebSocketClient({
-      config: {
+    const options: PolykeyClientOptions = {
+      nodePath: nodePath,
+      tokenBase: config.paths.tokenBase,
+      rpc: {
+        callTimeoutTime: config.defaultsSystem.rpcCallTimeoutTime,
+        parserBufferSize: config.defaultsSystem.rpcParserBufferSize,
+      },
```

Comment:
Unless they need to be set for the test to work, don't bother setting them. Less noise for the test.